packages/phone-number-privacy/signer/src/database/database.ts

```diff
@@ -1,16 +1,88 @@
 import knex from 'knex'
-import config, { DEV_MODE } from '../config'
+import Knex from 'knex/types'
+import logger from '../common/logger'
+import config, { DEV_MODE, SupportedDatabase } from '../config'
+import { ACCOUNTS_COLUMNS, ACCOUNTS_TABLE } from './models/account'
 
-const db = knex({
-  client: 'pg',
-  connection: config.db,
-  debug: DEV_MODE,
-})
+let db: Knex
+export async function initDatabase(doTestQuery = true) {
+  logger.info('Initializing database connection')
+  const { type, host, port, user, password, database, ssl } = config.db
 
-export function getDatabase() {
+  let dbConfig: any
+  let client: string
+  if (type === SupportedDatabase.Postgres) {
```

Comment:
Should this be a case insensitive comparison? No strong preference but wanted to be explicit.

Reply:
We lower case the value as it gets put in the config so that's covered